src/ducks/subject-viewer.js

```diff
@@ -184,9 +190,20 @@ const resetView = () => {
 };
 
 const togglePreviousMarks = () => {
-  return (dispatch) => {
+  return (dispatch, getState) => {
+    const annotations = getState().previousAnnotations.marks;
+    const variant = getState().splits.variant;
+    const consensusAnnotation = annotations.some(annotation => { return annotation.consensusReached; })
+    const previousAnnotationsPresent = consensusAnnotation || (annotations.length && variant === VARIANT_TYPES.COLLABORATIVE);
+
+    let shownMarks = (getState().subjectViewer.shownMarks + 1) % 3;
```

Comment:
Nice use of enumerated values! Ha, now I understand why you're using numbers instead of strings to represent each state.

One suggestion to improve any enumerated-value-sequencifier code is to do it like...
`... = (getState()subjectViewer.shownMarks + 1) % number_of(MARKS_STATE)`

This is only a minor suggestion though, because creating that number_of() function would probably take more brainspace than necessary, which is a given since JS doesn't have a natural `enum` [type](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/enum)